src/query/api/v1/handler/prometheus/native/read.go

```diff
@@ -137,6 +145,7 @@ func NewPromReadHandler(
 func (h *PromReadHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	timer := h.promReadMetrics.fetchTimerSuccess.Start()
 	fetchOpts, rErr := h.fetchOptionsBuilder.NewFetchOptions(r)
+	fetchOpts.IncludeExemplars = true
```

Comment:
Temporary forcing to always include exemplars in response but must be removed in the final version. Need to think how it should be passed from Grafana.